cypress/utils/dashboards/remove_workspace.js

```diff
@@ -7,7 +7,7 @@ export function removeSampleDataAndWorkspace(url, workspaceName) {
   describe('removing workspace/sampledata', () => {
     it('remove workspace', () => {
       cy.visit(`${url}/app/workspace_list`);
-      cy.openWorkspaceDashboard(workspaceName);
+      cy.osd.openWorkspaceDashboard(workspaceName);
       cy.getElementByTestId('toggleNavButton').eq(0).should('exist').click();
       cy.wait(3000);
```

Comment:
not specifc to this pr but how come we opt for the cy.wait instead of using something like `timeout`

example:
```
cy.get("#form-name", {timeout: 10000}).type("Mike Johnson")
```

https://docs.cypress.io/app/core-concepts/retry-ability#Timeouts

so we could put an increased timeout on the next line. cy.getElementByTestId (we might have to modify it if it's not available for that function). 

like 

```suggestion
  cy.getElementByTestId('collapsibleNavAppLink-workspace_detail', { timeout: 10000 }).should('exist').click();
```

then instead of an ambiguous wait time which could get flaky it will just keep retrying for the element until it sees it for 10000 ms

Reply:
Yeah, I agree we shouldn't use arbitrary waits given these are flaky and introduce race conditions. I'm a little weary of addressing in this PR since I'm not confident that I'll be able to address easily with the confidence that I won't break anything given I have somewhat limited context in our integration tests.